history-of-the-quakers/original/31-book-3-chapter-5.adoc

```diff
@@ -492,12 +499,13 @@ These arbitrary proceedings occasioning the prisoners, with just reason, to comp
 the executioner, as often as they attempted to speak, was ordered to stop their mouths,
 which he repeatedly did with a dirty cloth; and having a gag in his hand,
 endeavoured to gag John Crook and others.
-Upon this they cried out, Will you not give us leave to speak?
+Upon this they cried out, '`Will you not give us leave to speak?
 we except against some of the jury, as being our enemies,
 and of those who by force commanded us to be pulled out of our meetings,
 and carried us to prison without warrant or legal process; and must these be our judges?
-we except against them.
+we except against them.`'
 
+[.discourse-part]
```

Comment:
I wasn't sure if this should be [.discourse-part] too since it's all by itself.

Reply:
seems good to me. 👍